packages/vats/decentral-test-psm-config.json

```diff
@@ -0,0 +1,70 @@
+{
+  "bootstrap": "bootstrap",
+  "defaultReapInterval": 1000,
+  "snapshotInterval": 1000,
+  "vats": {
+    "bootstrap": {
+      "sourceSpec": "@agoric/vats/src/core/boot-psm.js",
+      "parameters": {
+        "anchorAssets": [
+          {
+            "denom": "ibc/usdc1234"
+          }
+        ],
+        "economicCommitteeAddresses": {
+          "gov1": "agoric1ldmtatp24qlllgxmrsjzcpe20fvlkp448zcuce",
+          "gov2": "agoric140dmkrz2e42ergjj7gyvejhzmjzurvqeq82ang",
+          "gov3": "agoric1w8wktaur4zf8qmmtn3n7x3r0jhsjkjntcm3u6h"
+        }
```

Comment:
Confirm that this is the only intended different from the main config.

Reply:
yep, and ultimately we'll patch up the denom in anchorAssets to axelar test once that's wired up